tests/test_users.py

```
@@ -0,0 +1,74 @@
+import pytest
+from httpx import AsyncClient
+from app.schemas import UserCreate, UserType
```

Ruff (0.11.9)

`app.schemas.UserCreate` imported but unused

Remove unused import

(F401)